logger/src/lib.rs

```diff
@@ -282,34 +248,23 @@ pub struct Logger {
     show_level: AtomicBool,
     show_file_path: AtomicBool,
     show_line_numbers: AtomicBool,
-    level_info: LevelInfo,
+    level: AtomicUsize,
     // Used in case we want to send logs to a FIFO.
-    log_fifo: Mutex<Option<PipeLogWriter>>,
+    log_buf: Mutex<Option<Box<dyn Write + Send>>>,
     // Used in case we want to send metrics to a FIFO.
-    metrics_fifo: Mutex<Option<PipeLogWriter>>,
+    metrics_buf: Mutex<Option<Box<dyn Write + Send>>>,
     instance_id: RwLock<String>,
     flags: AtomicUsize,
 }
 
-// Auxiliary function to get the default destination for some code level.
-fn get_default_destination(level: Level) -> Destination {
-    match level {
-        Level::Error => Destination::Stderr,
-        Level::Warn => Destination::Stderr,
-        Level::Info => Destination::Stdout,
-        Level::Debug => Destination::Stdout,
-        Level::Trace => Destination::Stdout,
-    }
-}
-
 // Auxiliary function to flush a message to a PipeLogWriter.
```

Comment:
This happens in a later commit. See https://github.com/firecracker-microvm/firecracker/pull/1257/commits/2d3123ceaa733eaa3daf9ce7a59d570d3dd4e2fa#diff-9b74e5bf0cb94b2a850c9aaf45020303R242. I chose to update comments and doc in a separate commit since there were plenty of them, sorry for the confusion.